implants/lib/eldritch/src/runtime/messages/set_callback_interval.rs

```diff
@@ -0,0 +1,41 @@
+use super::{Dispatcher, Transport};
+use anyhow::Result;
+use pb::{c2::Beacon, config::Config};
+
+/*
+ * SetCallbackIntervalMessage sets the callback interval in the dispatched config.
+ */
+#[cfg_attr(debug_assertions, derive(Debug, PartialEq))]
+#[derive(Clone)]
+pub struct SetCallbackIntervalMessage {
+    pub(crate) id: i64,
+    pub(crate) new_interval: u64,
+}
+
+impl SetCallbackIntervalMessage {
+    pub fn refresh_config(self, cfg: Config) -> Result<Config> {
+        let mut c = cfg.clone();
+        c.retry_interval = self.new_interval;
```

Comment:
I think there's a difference in callback_interval and retry_interval.
IIRC retry is - on error try again and callback is the normal callback.
I think we want callback here and or to merge the two into one variable - I usually set them to be the same.

Reply:
ok ill ignore retry_interval for now